blockchain/msgs.go

```diff
@@ -18,91 +12,3 @@ const (
 		BlockResponseMessagePrefixSize +
 		BlockResponseMessageFieldKeySize
 )
```

Comment:
Maybe we should move these constants into `bcproto` too? Especially since they need to be kept in sync with code there.

Reply:
Sure.